.github/workflows/helm-release.yaml

```diff
@@ -14,43 +14,20 @@ jobs:
         with:
           fetch-depth: 0 # Important for semantic-release to analyze all commits
 
-      - name: Configure Git
+      - name: Install Helm
+        if: steps.semantic.outputs.new_release_published == 'true'
```

Comment:
Can this even work ?
Semantic release get's called after this step ?

Reply:
lol thx